src/main/java/nus/climods/logic/LogicManager.java

```diff
@@ -36,13 +39,21 @@ public LogicManager(Model model, Storage storage) {
     }
 
     @Override
-    public CommandResult execute(String commandText) throws CommandException, ParseException {
+    public CommandResult execute(String commandText) throws CommandException, ParseException, IOException {
```

Comment:
I think we can probably have IO errors as `StorageException` instead? I guess it is just to be specifc wdyt?

Reply:
sounds good